proposals/61:lo-bundle-cost/Makefile

```diff
@@ -0,0 +1,13 @@
+
+lower-bundle-cost.json:
+	jq '.tx.body.messages[0].content | .changes[0].value |= fromjson' tx-61.json >$@
```

Comment:
On first read I thought this was able to build dynamically from the ARCHIVE_NODE.

Please comment how this Makefile is intended to be used.

Incidentally, I suggest that `lower-bundle-cost.json` be produced during the build since it's a fast local operation to create from `tx-61.json`